packages/kit/src/runtime/server/index.js

```diff
@@ -96,6 +96,10 @@ export async function respond(request, options, state) {
 		}
 	}
 
+	if (!route && url.pathname === '/favicon.ico') {
+		return new Response('not found', { status: 404 });
+	}
+
```

Comment:
we decided just to remove special handling of `/favicon.ico` altogether

```suggestion
	if (!route && url.pathname === '/favicon.ico') {
		return new Response('not found', { status: 404 });
	}

```